cli/src/commands/log.rs

```diff
@@ -147,6 +147,11 @@ pub(crate) fn cmd_log(
         }
         expression
     };
+    let prio_revset = settings
+        .get_string("revsets.log-graph-prioritize")
+        .optional()?
```

Comment:
I think the default `@` can be defined in config/revsets.toml.

Or do you want to set prioritized branch in addition to `@`? In that case, which revset should be displayed first? We might have to add a list of prioritized revsets.

Reply:
Good point, I originally tried adding it addition, but I don't think it made a big difference from my testing, and a single revset was always enough for me.

Reply:
Maybe it should just expect that the revset only contains a single revision. I'm not sure renderdag does much with multiple prioritizations, so usually `coalesce(1, 2, 3)` is what you want instead of `1 | 2 | 3`.
Especially since the order of the revset is not defined.

Reply:
> Maybe it should just expect that the revset only contains a single revision.

Yes, the set size should usually be 0 or 1, but I wouldn't add extra check for that.

We might want to add some sorting or ordered union revset functions that can be used only in certain context, but I'm not sure.